src/main/java/Storage.java

```diff
@@ -0,0 +1,6 @@
+public class Storage {
```

Comment:
Storage needs to read and write tasks. Serves one purpose only. Consider using FileWriter, PrintWriter classes.